test/testdata/.gitignore

```diff
@@ -0,0 +1,3 @@
+.pyc
```

Comment:
you can move this to the main `gitignore`

Reply:
I'll remove them here! I seem to remember I added them because a test failed that wanted to see them there.